lib/matplotlib/patches.py

```diff
@@ -1037,13 +1037,24 @@ def _update_path(self):
                              f"`len(values) = {self._values.size}` and "
                              f"`len(edges) = {self._edges.size}`.")
         verts, codes = [], []
-        for idx0, idx1 in cbook.contiguous_regions(~np.isnan(self._values)):
+
+        _valid_values = np.isnan(self._values)
```

Comment:
```suggestion
        nan_mask = np.isnan(self._values)
```
(Follow-up of the logic inversion).